.gitignore

```diff
@@ -395,4 +395,6 @@ FodyWeavers.xsd
 *.msp
 
 # JetBrains Rider
-*.sln.iml
\ No newline at end of file
+*.sln.iml
+
+appsettings.json
```

Comment:
appsettings must not be in gitignore. It's impossible to add new sections to them if needed